docs/procedures/rigid_waist_shift.md

```diff
@@ -0,0 +1,77 @@
+# LHC Local Coupling Corrections with a Rigid Waist Shift
+
+!!! note ""
+    Please keep in mind the [general checks](general_checks.md) for measurements.
+
+??? info "The Procedure in Short"
+
+      This methods aims to find the MQSX correction settings that would minimize betatron coupling and its impact on beam size *at IP*.
+      The method breaks the symmetry of the optics in the IR and forces local coupling RDTs to leak thoughout the machine, which makes them measurable through the $|C^{-}|$.
+
+
+      After global corrections are done and trimmed in the machine, one applies a rigid waist shift in a given IR and scans the colinearity knob for the value that minimises the $|C^{-}|$.
+      These settings, when taking away the rigid waist shift, will minimize local coupling and its impact at IP.
+
+## Preliminary Setup
+
+- [ ] <details class="nodeco"><summary>Do Global Corrections</summary>
+      <p> This procedure needs global corrections to be trimmed in the machine first, so optics and *global* coupling should be taken care of beforehand.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>**Optional:** Scan the Colinearity Knob to Check Conditions</summary>
+      <p> If time allows, ideally we would scan the colinearity knob to ensure we see very small variations of the $|C^{-}|$.
+      If strong variations are noticed, then the expected conditions for the procedure are not met: either the phase advance between left and right MQSXs is off, or the $\sqrt{\beta_x \beta_y}$ is significantly wrong at these elements.
+      </p></details>
+
+## Procedure Per IP
+
+One should do this for earch of IP1, IP2, IP5 and IP8 which are the ones requiring local corrections.
+Keep in mind that this does beam 1 and 2 at the same time.
+
+!!! warning "Orbit Feedback"
+      Please remember to **always** keep the orbit feedback system ON during this procedure.
+
+- [ ] <details class="nodeco"><summary>Trim in the Waist Shift Knob</summary>
+      <p> Trim the prepared knob in the machine, for a certain direction (waist left/right of IP).
+      Remember that this affects both beams at the same time.
```

Comment:
Maybe add a section **Preparation** before where the knob details are described. E.g. knob uses MQX, one unit changes waist by X cm or one unit trims the quads by X %. Depending on what one unit is, this also affects then the next sentence, if one unit is the maximum or not.

Reply:
Actually just read the explanation further down, comment redundant.